imports/plugins/core/orders/server/no-meteor/util/getSurchargesForGroup.js

```diff
@@ -0,0 +1,52 @@
+import xformOrderGroupToCommonOrder from "./xformOrderGroupToCommonOrder";
+
+/**
```

Comment:
You mention we don't do `surcharges` per group in other changes here: https://github.com/reactioncommerce/reaction/pull/5027/files#diff-8ce970cbd524f077298082ab81660a4bR107

Is this still the case even with this new code?

Reply:
My comment should be clearer. We calculate them per group but they're only stored on the whole order, so we merge them all into one array. That's still true.

Reply:
It's possible that's not correct, but I'd have to add `surcharges` to fulfillment group schema if we want to change it. Seems like something we can figure out later, when surcharges are more in use.